lib/cisco_node_utils/cmd_ref/snmpnotification.yaml

```diff
@@ -0,0 +1,11 @@
+# snmpnotification
+---
+enable:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/^(no )?snmp-server\senable\straps\s%s\s?$/'
```

Comment:
Couple of nit picks here.

1) Remove the explicit `\s` checks for white space.
2) Please use key-value wild cards for `config_get_token` instead of `%s`